src/taipy/core/config/scenario_config.py

```diff
@@ -319,3 +336,17 @@ def _configure_default(
         )
         Config._register(section)
         return Config.sections[ScenarioConfig.name][_Config.DEFAULT_KEY]
+
+    def add_sequences(self, sequences: Dict[str, List[TaskConfig]]):
+        # TODO: How about user just provides task config id
```

Comment:
I would not accept ids. It just adds some complexity to me, for very little added value.